java/src/FilterData.java

```diff
@@ -16,16 +16,20 @@ private String getPath(String fileName) {
 
     private void run() {
 
+        //Input filename
         Scanner sc = new Scanner(System.in); // System.in is a standard input stream.
-        System.out.print("Enter a filename (without .txt): ");
+        System.out.println("Enter a filename (without .txt): ");
+        System.out.println("avicii or catData");
```

Comment:
Not clear what you mean with this: if you are giving a suggestion to file names be more clear.

Reply:
This is intended to give the user suggestions for what is a possible entry at this stage.

Reply:
Is it unclear for the user or for a developer?

Reply:
Maybe just add "possible file names to choose from: avicii or catData" and it will be clearer for users and developers

Reply:
Ok!